pymongo/read_preferences.py

```diff
@@ -151,6 +151,11 @@ def tag_sets(self) -> _TagSets:
         set, ``{}``, means "read from any member that matches the mode,
         ignoring tags." MongoClient tries each set of tags in turn
         until it finds a set of tags with at least one matching member.
+        For example, to only send a query to an analytic node::
+
+           Nearest(tag_sets=[
+              {"node":"analytics"}
+           ])
```

Comment:
Make this one line like below: `Nearest(tag_sets=[{"node":"analytics"}])`

Can you also add a similar SecondaryPreferred example?